app/test_split_integer.py

```diff
@@ -1,21 +1,91 @@
+import pytest
+
 from app.split_integer import split_integer
 
 
-def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+@pytest.mark.parametrize(
+    "value,number_of_parts,expected_result",
+    [
+        (8, 1, [8]),
+        (6, 2, [3, 3]),
+        (17, 4, [4, 4, 4, 5]),
+        (32, 6, [5, 5, 5, 5, 6, 6])
+    ],
+)
+def test_sum_of_the_parts_should_be_equal_to_value(
+        value: int,
+        number_of_parts: int,
+        expected_result: list[int, ...]
+) -> None:
+    assert split_integer(value, number_of_parts) == expected_result
 
 
-def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+@pytest.mark.parametrize(
+    "value,number_of_parts",
+    [
+        (8, 1),
+        (6, 2),
+        (17, 4),
+        (32, 6)
+    ],
+)
+def test_should_split_into_equal_parts_when_value_divisible_by_parts(
+        value: int,
+        number_of_parts: int
+) -> None:
+    assert (
+        split_integer(value, number_of_parts)
+        == [value // number_of_parts] * number_of_parts
+    )
```

Comment:
This test case checks if the function splits the value into equal parts when the value is divisible by the number of parts. However, this is not always the case. For example, if the value is 8 and the number of parts is 3, the function should return [2, 3, 3], not [2, 2, 2]. Consider revising the test case to reflect this.